task/06-conditions-n-loops-tasks.js

```diff
@@ -119,7 +134,12 @@ function isTriangle(a,b,c) {
  *  
  */
 function doRectanglesOverlap(rect1, rect2) {
-    throw new Error('Not implemented');
+    if (rect1.top < rect2.top + rect2.width &&
+        rect1.left < rect2.left + rect2.height &&
+        rect2.top < rect1.top + rect1.width &&
+        rect2.left < rect1.left + rect1.height)
+        return true;
+    else return false;
```

Comment:
Please, solve this task using only one return
https://javascript.info/logical-operators